docs/guide/overview.asciidoc

```diff
@@ -38,6 +36,42 @@ This data is buffered for a short period and sent on to APM Server.
 It listens on port 8200 by default and receives data from agents through a JSON HTTP API.
 Then it creates documents from that data and stores them in Elasticsearch.
 
-To visualize the data after it's sent to Elasticsearch,
-you can use the the dedicated APM UI bundled in X-Pack,
-or the pre-built open source Kibana dashboards that can be loaded directly via the {kibana-ref}/apm-getting-started.html[APM Kibana UI].
\ No newline at end of file
+*Elasticsearch* is a highly scalable open-source full-text search and analytics engine.
+It allows you to store, search, and analyze big volumes of data quickly and in near real time.
+
+*Kibana* is an open source analytics and visualization platform designed to work with Elasticsearch.
+You use Kibana to search, view, and interact with data stored in Elasticsearch indices.
+You an use Kibana to visualize APM data by utilizing the dedicated APM UI bundled in Basic license,
+or the pre-built, open source,
+Kibana dashboards that can be loaded directly via the {kibana-ref}/apm-getting-started.html[APM Kibana UI].
+
+[[concepts]]
+[float]
+=== APM Concepts
+
+*Agents*
+APM Agents capture different types of information from within their instrumented applications, known as events. Events can be <<transactions,`Transactions`>>, <<transaction-spans,`Spans`>>, and <<errors,`Errors`>>.
+These events are then streamed to a single endpoint in the APM Server which sorts and processes the events. 
+
+* <<transactions,`Transactions`>> - A transaction describes an event captured by an Elastic APM agent instrumenting a service. For example: an HTTP request, or a background job.
```

Comment:
I fear to open up this can of worms, but I like to think of transactions as just a special kind of span with extra metadata associated with them. When demoing Elastic APM, I find it much easier to describe transactions this way.

Also, in "A transaction describes an event captured by an Elastic APM agent instrumenting a service" you could swap out the word "transaction" with "span" and it would still be correct - so I don't think this really describes what make transactions different from spans. I usually describe transactions as the highest level of work you're measuring within a service. E.g. serving an HTTP request or running a specific background job. All work performed as a result of this high level transaction are measured as spans - e.g. database queries necessary to serve the HTTP request etc.

Reply:
This is great! I've mashed this with some of the notes I took in our conversation on Friday. I think _all_ of this might be too much to add to the overview, but what I leave out should be incorporated in the APM Data Model.